src/internals/eventHandler.ts

```diff
@@ -0,0 +1,38 @@
+import { Collection } from "discord.js";
+import { loadFiles } from "../utils/loadFiles";
+import { join } from "path";
+import type { Client, ClientEvents } from "discord.js";
+import type { DiscordEvent } from "../types/DiscordEvent";
+
+const events = new Collection<string, DiscordEvent>();
```

Comment:
what is the point of putting `events` outside? why can't events be declared via line 10?

Reply:
the collection was defined outside with future needs for exporting the event in mind